crypto/store/loader_file.c

```diff
@@ -845,7 +845,8 @@ static OSSL_STORE_LOADER_CTX *file_open(const OSSL_STORE_LOADER *loader,
             return NULL;
         }
 
-        if (stat(path_data[i].path, &st) < 0) {
+        if (path_data[i].path[0] != '\0' && /* TODO remove when #11756 is merged */
+            stat(path_data[i].path, &st) < 0) {
```

Comment:
nit : && logical operator should be at the begin of this statement-line.

Reply:
The condition and the `&&` following it were part of a temporary fix as indicated by the comment following it and by the message of the respective commit 008c493049e13a81409ddfafef0ceda1449d1d76: 
```
TMP FIX until #11756: Make OSSL_STORE_open() interpret empty uri () as <stdin>
```
It turns that the fix is not really needed, so I've reverted that commit.